t04_refactored.py

```diff
@@ -338,8 +338,52 @@ def team_12_adv():
 
 
 def team_13_adv():
-    pass
-    # TODO Add your code here
+    """
+    Andy Hill
+    Nyan Lin Zaw
+    https://docs.google.com/document/d/1-lO7SPynvH8GxfrMS5xBYxhiu9YZs-9oWcI3YCdIKTA/edit
+    :return: none
+    """
+
+    from time import sleep
```

Comment:
This is not resolved. This line should be removed. The same import exists at the beginning of this file.